### ISSUE TITLE
Colorblind safe alert colors

### ISSUE THREAD

**Issue by mapsam** [18F/web-design-standards#811]
Hello! Thank you for all of your hard work on this. :clap: 

**Description**   

I was perusing the [alert components](https://playbook.cio.gov/designstandards/alerts/) and noticed the color scheme, although quite elegant, is not terribly colorblind friendly. Personally I am unable to tell the difference between `success` and `warning`, which could complicate things. Since they are next to each other in the style guide, I can discern a slight difference, but if they were alone I would have no idea what color it was. The icons are helpful (yay!), but if color is the primary sense of understanding it seems important to make sure they work for everyone!

As a _deuteranope_ (red green colorblind) I can quite easily distinguish between the `alert` red and the `success`, but I'm unable to tell if `success` is green or yellow.

**Ask**   

I'd love to see a more prominent green, as they both look slightly yellow - and don't send a sense of accomplishment.

Below is a current photo and a suggested photo with an updated green, as well as some color pairings with some thoughts.

**Current**

<img width="910" alt="screen shot 2015-10-08 at 2 06 05 pm" src="https://cloud.githubusercontent.com/assets/1943001/10375503/ceaf186c-6dc6-11e5-8e18-6b872a4a014a.png">

**Suggested** update with green hex `#c0efc5` (note: I didn't spend a TON of time finding this green, wouldn't recommend taking it as-is :wink:)

<img width="905" alt="screen shot 2015-10-08 at 2 19 14 pm" src="https://cloud.githubusercontent.com/assets/1943001/10375627/a5bdd3ac-6dc7-11e5-8041-c762feddca67.png">

Some pairings with _Notes from a Deuteranope_ - (note: these are my un-scientific opinions, cannot back them up with anything but my own thoughts)

<img width="798" alt="screen shot 2015-10-08 at 2 37 40 pm" src="https://cloud.githubusercontent.com/assets/1943001/10376113/2f2d3766-6dca-11e5-8da1-4eea7ad05d06.png">

Thanks!

**Comment by MsWebGirl** [comment]
++!

**Comment by mollieru** [comment]
Thanks! We'll take a look at this and come up w a solution that is more visually accessible. 

@angel can you take a look here? Happy to help you out!

**Comment by msecret** [comment]
@rtwell as new visual designer, do you have any thoughts here?

**Comment by rtwell** [comment]
These alerts are compliant: we use more than color alone to convey information about the alert. _In theory_ users will not see those alerts side by side, so discerning between them is, from a practical standpoint, irrelevant. That said, there are folks that DO see them side by side, and it would be great if we could accommodate them!

I would recommend:

1. Refine the design of our alerts to include a darker slab of color on the far left, whilst preserving the light tones for legibility, perhaps something like this: 
![image](https://user-images.githubusercontent.com/12564977/28421734-a65591e6-6d2b-11e7-8f5c-94aa735f7269.png)

2. Update our color palette with tones of color that are more easily distinguished by everyone

**Comment by maya** [comment]
@mapsam can you let us know if the new alert design works better for you (darker color border on the left)? Like @rtwell mentioned, we convey the state in other ways with an icon and text so it should be discernible even w/o color.

**Comment by mapsam** [comment]
Hey, these look really great! 👍 from this Deuteranope.

**Comment by maya** [comment]
Wonderful, glad these are working for you! 🎉
